test/chromium/coverage.spec.js

```diff
@@ -135,11 +135,11 @@ describe('CSSCoverage', function() {
       {start: 17, end: 38}
     ]);
   });
-  it('should work with complicated usecases', async function({page, server}) {
+  it('should work with complicated usecases', async function({page, server, golden}) {
     await page.coverage.startCSSCoverage();
     await page.goto(server.PREFIX + '/csscoverage/involved.html');
     const coverage = await page.coverage.stopCSSCoverage();
-    expect(JSON.stringify(coverage, null, 2).replace(/:\d{4}\//g, ':<PORT>/')).toBeGolden('csscoverage-involved.txt');
+    expect(JSON.stringify(coverage, null, 2).replace(/:\d{4}\//g, ':<PORT>/')).toBeGolden(golden('csscoverage-involved.txt'));
```

Comment:
why can't `expect().toBeGolden()` encapsulate logic of `golden()` and enable clients pass just string name as before? (`toBeGolden(golden(` seems a bit ugly)

Reply:
`expect` is a global function, while `golden` state is test-specific. Currently, when running two tests from different browsers in parallel, only one of the tests can use the correct golden. Further improvements on capturing per-test output will also rely on test attribution.